tensorflow/tools/ci_build/windows/gpu/pip/build_tf_windows.sh

```diff
@@ -46,7 +46,7 @@ run_configure_for_gpu_build
 
 clean_output_base
 
-bazel build -c opt --config=win-cuda $BUILD_OPTS tensorflow/tools/pip_package:build_pip_package || exit $?
+bazel build -c opt $BUILD_OPTS tensorflow/tools/pip_package:build_pip_package || exit $?
```

Comment:
Without the "--config" flag, where/how do we select GPU config for the build?
Are we sure that the built binaries still use GPU?